ios/Classes/SmileIDBiometricKYC.swift

```diff
@@ -0,0 +1,101 @@
+import Flutter
+import UIKit
+import SmileID
+import SwiftUI
+
+class SmileIDBiometricKYC : NSObject, FlutterPlatformView, BiometricKycResultDelegate {
+    private var _view: UIView
+    private var _channel: FlutterMethodChannel
+    private var _childViewController: UIViewController?
+
+    static let VIEW_TYPE_ID = "SmileIDBiometricKYC"
+
+    init(
+        frame: CGRect,
+        viewIdentifier viewId: Int64,
+        arguments args: [String: Any?],
+        binaryMessenger messenger: FlutterBinaryMessenger
+    ) {
+        _view = UIView()
+        _channel = FlutterMethodChannel(
+            name: "\(SmileIDBiometricKYC.VIEW_TYPE_ID)_\(viewId)",
+            binaryMessenger: messenger
+        )
+        _childViewController = nil
+        super.init()
+        let screen = SmileID.biometricKycScreen(
+            idInfo: IdInfo(
+                country:  args["userId"] as? String ?? "",
+                idType:  args["idType"] as? String,
+                idNumber:  args["idNumber"] as? String,
+                firstName:  args["firstName"] as? String,
+                middleName:  args["middleName"] as? String,
+                lastName:  args["lastName"] as? String,
+                dob:  args["dob"] as? String,
+                bankCode:  args["bankCode"] as? String,
+                entered:  args["entered"] as? Bool
+            ),
+            userId: args["userId"] as? String ?? "user-\(UUID().uuidString)",
```

Comment:
Let's use the helper methods that are available in the iOS SDK
```suggestion
            userId: args["userId"] as? String ?? generateUserId(),
```

Reply:
Done